### PR TITLE
Feat like video with UI

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull Request , the reviewer would be able to use the like video feature of MI Stream. 
In order to use this feature the reviewer would first need to login into the app followed by that he can like the video from 2 places.
1) Either at the explore page where each video would have a menu dropdown showing the reviewer to add a video or remove it from a collection
2) By going to the video page of the respective video and then adding it to like dislikes by toggling on the thumbs up icon itself

To view the list of liked videos, the reviewer need to go to the liked page itself to check the videos which he has liked and use the dropdown available on each video to remove a particular video.
So in this Pull request the reviewer would have to review 2 major pages:
1) Empty Page (if there are no liked videos)
2) Likes Page

And along with that they would have to see 2 major components that helped in building the feature:
1) Video Menu Options
2) Collection Card [that represents the component being used for the videos used in the Liked Videos Section of the liked videos page]

And the stylesheets of the above mentioned components and pages

### CODE REVIEW

File: src/pages/EmptyPage/EmptyPage.module.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.empty_collection_container {
+  margin: 1rem;
+}
+
+.empty_collection_contents {
+  gap: 1rem;
```
Comment (sameep-s) [inline src/pages/EmptyPage/EmptyPage.module.css:6]: Just a suggestion gap is not supported in some browsers try achieving this with flex only,

File: src/components/VideoSuggestionCard/VideoSuggestionCard.jsx [hunk 2]
```diff
@@ -20,11 +22,18 @@ const VideoSuggestionCard = ({ videoReceived }) => {
       <div className={`${styles.video_suggestion_card_contents} g-flex-column`}>
         <div className={`g-flex-row`}>
           <p className={`${styles.video_suggestion_card_title}`}>{title}</p>
-          <span
+          {/* <span
             className={`material-icons g-flex-center ${styles.video_suggestion_card_more_options}`}
```
Comment (sameep-s) [inline src/components/VideoSuggestionCard/VideoSuggestionCard.jsx:26]: can remove these comments

File: src/pages/EmptyPage/EmptyPage.module.css (hunk 1 above)
Comment (dev-enforced) [reply src/pages/EmptyPage/EmptyPage.module.css:6]: Yes I have used flex gap property itself . I used utility classes of flex itself for that div

File: src/components/VideoSuggestionCard/VideoSuggestionCard.jsx (hunk 2 above)
Comment (dev-enforced) [reply src/components/VideoSuggestionCard/VideoSuggestionCard.jsx:26]: Sure doing it right away!!

**Review by sameep-s — APPROVED** [review]
Things i liked:
- Good Naming convention throughout.
- Clean Code.
- Good logic.

Some Suggestion:
- You are using gap in a lot of places and it is not fully supported in all browser so try using flex only.
- Remove commented code.
- You are using a css-library, and what looks like styled components which can be confusing and bugs can be hard to find as you would be looking at two places.

other than some minor suggestions code quality is good.

**Comment by ankit9015** [comment]
Overall Code is clean and looks good to merge

**Pull request merged**